tests/integration/KeyMappingManager/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <Core/Resources/Resources.hpp>
+#include <Core/Utils/StdFilesystem.hpp>
+#include <Gui/Utils/KeyMappingManager.hpp>
+
+#include <QtGlobal>
+
+using namespace Ra;
+using namespace Ra::Gui;
+using Idx = KeyMappingManager::KeyMappingAction;
+using Ctx = KeyMappingManager::Context;
+
+int main( int argc, char** argv ) {
+
+    /// need a base dir in argv[1]
+    if ( argc != 2 ) { return -1; }
+    std::string base {argv[1]};
+
+    QSettings settings( "IntegrationTests", "KeyMappingManager" );
```

Comment:
```suggestion
    QSettings settings( "RadiumIntegrationTests", "KeyMappingManager" );
```
?

Reply:
Fine, I add it to both unit and integration tests.